content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -78,6 +81,8 @@ configured as a 6-pack, a 12-pack, or a case of 36. Each flavor of soda would ne
 can packaging configured on the individual product since packagings are product specific, not
 generic.
 
+Create different packaging for a product
```

Comment:
I was thinking we could add more sections for readers to jump to on the `On this page` section
![image](https://user-images.githubusercontent.com/23611711/230510272-bfa5bbcd-2a6c-4314-af49-9a590385123b.png)

Reply:
Great!